### ISSUE TITLE
Custom dispatcher using FQCN support

### ISSUE THREAD

**Issue by rpankajakshan** [kamon-io/kamon-akka#20]
I have the following snippet in my application config to setup a custom dispatcher using FQCN

s3-dispatcher {
      type = "Dispatcher"
      executor = "com.a.b.s3.S3ThreadPoolExecutorConfigurator"
      maxConnections = 500
    }

s3-dispatcher does not show up in the metrics. Please could you support this?

ref - https://doc.akka.io/docs/akka/current/scala/dispatchers.html

This is on 
kamon 0.6.7
akka 2.5.6
scala 2.12

**Comment by gmalouf** [comment]
@rpankajakshan can you confirm you are pulling in the kamon-akka-2.5 dependency (not 2.4)?  I ran into an issue with kamon-akka-http pulling in kamon-akka-2.4 where I was only seeing one dispatcher; excluding the 2.4 version and explicitly including the 2.5 one fixed it for me.

**Comment by rpankajakshan** [comment]
@gmalouf yes i can confirm that i am using kamon-akka-2.5. Dependencies look like this

  val kamonJars: Seq[ModuleID] =
    Seq(
      "io.kamon" %% "kamon-akka-http" % "0.6.8" exclude("io.kamon", "kamon-akka-2.4_2.12"),
      "io.kamon" %% "kamon-akka-2.5" % "0.6.8",
      "io.kamon" %% "kamon-core" % "0.6.7",
      "io.kamon" %% "kamon-system-metrics" % "0.6.6",
      "com.github.milenkovicm" %% "kamon-prometheus" % "0.3.0"
    )

i also looked into the code and FQCN support for dispatchers are missing
